docs/standards/lsp-background/erc725.md

```diff
@@ -14,69 +13,78 @@ sidebar_position: 1
-|        4         | [`DELEGATECALL`](https://eips.ethereum.org/EIPS/eip-7) | run the function from another contract, but use and update the storage of the current contract (= persist `msg.sender` and `msg.value`) |
+| Operation number |                     Operation type                     | Description                                                                                                         |
+| :--------------: | :----------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------ |
+|        0         |          [`CALL`](https://www.evm.codes/#f1)           | Transfer native tokens or calls smart contract functions.                                                           |
+|        1         |         [`CREATE`](https://www.evm.codes/#f0)          | Create a new smart contract based on the contract address and nonce.                                                |
+|        2         |  [`CREATE2`](https://eips.ethereum.org/EIPS/eip-1014)  | Create a new smart contract based on the contract address, bytecode and the salt. The address can be predetermined. |
+|        3         | [`STATICCALL`](https://eips.ethereum.org/EIPS/eip-214) | Calls another smart contract while disallowing any modification to the state during the call.                       |
+|        4         | [`DELEGATECALL`](https://eips.ethereum.org/EIPS/eip-7) | Runs the function from another contract, but use the context of the current contract.                               |
+
+## ERC725Y Generic Data Key/Value Store
+
+Using smart contracts to store data on the blockchain can be beneficial when that data will be used or referenced by other smart contracts.
+
+One limitation of this approach is that once a smart contract is deployed with a specific set of variables containing data, it cannot be modified to include new variables. This can be a problem if additional data needs to be stored in the future, as it would require deploying a new version of the contract.
+
+For example, if a token contract is deployed with variables for the token name, symbol, and decimals, it would not be possible to add new variables for the creators or future collaborators.
 
+What is needed is a dynamic way to attach data to a contract even after it has been deployed.
 
+### Specification
 
-### ERC725Y - Generic Data Key/Value Store
+ERC725Y solves this problem by standardizing a mapping of **data keys** to **data values** to store data dynamically, and to have the ability to add or remove data across time without the need of redeploying the contract.
+
+Instead of representing the variable with a specific name such as `age`, or `name`, etc .. They are referenceed with `bytes32`. Same as solidity treat variables under the hood.
```

Comment:
Not sure if we can states "data key" at this point?

```suggestion
Instead of representing the variable with a specific name such as `age`, or `name`, etc .. They are referenced as `bytes32` data key. Same as solidity treat variables under the hood.
```

If you feel not, you can discard this comment :)